opentracing_instrumentation/client_hooks/strict_redis.py

```diff
@@ -98,6 +98,8 @@ def execute_command(self, cmd, *args, **kwargs):
 
         # for certain commands we'll add extra attributes such as the redis key
         for tag_key, tag_val in getattr(self, '_extra_tags', []):
+            if isinstance(tag_val, basestring):
+                tag_val = tag_val.encode('string_escape')
```

Comment:
I think that makes more sense actually. Some of the keys are binary by design though, right? I don't know as much about the wire protocol as you.